OpenMcdf.Ole/OlePropertiesContainer.cs

```diff
@@ -36,6 +48,7 @@ public OlePropertiesContainer(int codePage, ContainerType containerType)
         };
 
         ContainerType = containerType;
+        FMTID0 = FmtIdFromContainerType(containerType);
```

Comment:
There was a previous suggestion to add a constructor that allows users to specify the fmtid to use rather than inferring it such that values the library doesn't know about can be used, but I don't need tht ability myself so I never spent any time on it.
Could be done later if useful though